fpdf/fpdf.py

```diff
@@ -1765,49 +1770,100 @@ def add_font(self, family, style="", fname=None, uni="DEPRECATED"):
         if fontkey in self.fonts or fontkey in self.core_fonts:
             warnings.warn(f"Core font or font already added '{fontkey}': doing nothing")
             return
+
         for parent in (".", FPDF_FONT_DIR):
             if not parent:
                 continue
+
             if (Path(parent) / fname).exists():
                 ttffilename = Path(parent) / fname
                 break
         else:
             raise FileNotFoundError(f"TTF Font file not found: {fname}")
 
-        # include numbers in the subset! (if alias present)
-        # ensure that alias is mapped 1-by-1 additionally (must be replaceable)
-        sbarr = "\x00 "
-        if self.str_alias_nb_pages:
-            sbarr += "0123456789"
-            sbarr += self.str_alias_nb_pages
+        # font tools
+        ft = ttLib.TTFont(ttffilename)
```

Comment:
```suggestion
        font = ttLib.TTFont(ttffilename)
```
`pylint` may not be happy with 2-letters variable names, which are not always very readable

Reply:
Yes I agree, actually I used that name for fast use and forgot to expand it later